application/helpers/mpdf_helper.php

```diff
@@ -111,7 +103,8 @@ function pdf_create(
             }
         }
 
-        $archived_file = UPLOADS_ARCHIVE_FOLDER . date('Y-m-d') . '_' . $filename . '.pdf';
+        //$archived_file = UPLOADS_FOLDER . 'archive/' . date('Y-m-d') . '_' . $filename . '.pdf'; // No date in the filename!
+        $archived_file = UPLOADS_FOLDER . 'archive/' . $filename . '.pdf';
```

Comment:
Same. Revert please and remove commented code. There is a reason for using the date in archived invoices.